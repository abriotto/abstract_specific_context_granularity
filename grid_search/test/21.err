  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:01, 67.62it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:00<00:01, 69.87it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:00<00:01, 61.58it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:00<00:01, 55.56it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:00<00:01, 55.10it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:00<00:01, 55.11it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:00<00:00, 67.05it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:00<00:00, 71.61it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:01<00:00, 67.16it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:01<00:00, 65.19it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:01<00:00, 63.69it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:01<00:00, 60.30it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 66.22it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 63.76it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:00, 59.01it/s] 52%|#################################################################################################2                                                                                         | 13/25 [00:00<00:00, 56.37it/s] 84%|#############################################################################################################################################################                              | 21/25 [00:00<00:00, 63.13it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 59.76it/s]
