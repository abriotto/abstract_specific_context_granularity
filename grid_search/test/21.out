# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.712873637676239, "acc": 0.490180641412735, "length": 2.5685195922851562, "mode": "train", "epoch": 1}
{"loss": 0.7009342908859253, "acc": 0.505586564540863, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6985830664634705, "acc": 0.48977428674697876, "length": 1.0097192525863647, "mode": "train", "epoch": 2}
{"loss": 0.6995500922203064, "acc": 0.4962756633758545, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6942786574363708, "acc": 0.5118474960327148, "length": 1.0005245208740234, "mode": "train", "epoch": 3}
{"loss": 0.6970453262329102, "acc": 0.49627557396888733, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6958615183830261, "acc": 0.51838219165802, "length": 1.0007538795471191, "mode": "train", "epoch": 4}
{"loss": 0.6967781782150269, "acc": 0.49813786149024963, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6980488300323486, "acc": 0.5149512887001038, "length": 1.0005255937576294, "mode": "train", "epoch": 5}
{"loss": 0.7018717527389526, "acc": 0.4869646430015564, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6977213621139526, "acc": 0.5024915337562561, "length": 1.0003544092178345, "mode": "train", "epoch": 6}
{"loss": 0.6958739757537842, "acc": 0.49720674753189087, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6931647658348083, "acc": 0.5246105790138245, "length": 1.00030517578125, "mode": "train", "epoch": 7}
{"loss": 0.7018585205078125, "acc": 0.4841713607311249, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6920279264450073, "acc": 0.5196319818496704, "length": 1.0006802082061768, "mode": "train", "epoch": 8}
{"loss": 0.6949915885925293, "acc": 0.4944134056568146, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6918313503265381, "acc": 0.52616286277771, "length": 1.0020124912261963, "mode": "train", "epoch": 9}
{"loss": 0.6976287364959717, "acc": 0.505586564540863, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.694010317325592, "acc": 0.52647864818573, "length": 1.0004198551177979, "mode": "train", "epoch": 10}
{"loss": 0.6973432302474976, "acc": 0.49068906903266907, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6931481957435608, "acc": 0.5165100693702698, "length": 1.0004403591156006, "mode": "train", "epoch": 11}
{"loss": 0.6977026462554932, "acc": 0.5000000596046448, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6934515237808228, "acc": 0.52522212266922, "length": 1.0022202730178833, "mode": "train", "epoch": 12}
{"loss": 0.6975886821746826, "acc": 0.5046554803848267, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6914375424385071, "acc": 0.5235382318496704, "length": 1.2667967081069946, "mode": "train", "epoch": 13}
{"loss": 0.708040177822113, "acc": 0.543554961681366, "length": 3.5139665603637695, "mode": "test", "epoch": 13}
{"loss": 0.6580097675323486, "acc": 0.6108047366142273, "length": 2.2801945209503174, "mode": "train", "epoch": 14}
{"loss": 0.6402642130851746, "acc": 0.6360430121421814, "length": 2.8770949840545654, "mode": "test", "epoch": 14}
{"loss": 0.6190054416656494, "acc": 0.6569899916648865, "length": 2.659700870513916, "mode": "train", "epoch": 15}
{"loss": 0.6479862928390503, "acc": 0.6286662220954895, "length": 2.692737340927124, "mode": "test", "epoch": 15}
{"loss": 0.5987936854362488, "acc": 0.6812946200370789, "length": 2.62221360206604, "mode": "train", "epoch": 16}
{"loss": 0.6254948377609253, "acc": 0.6541367173194885, "length": 2.9273743629455566, "mode": "test", "epoch": 16}
{"loss": 0.5778513550758362, "acc": 0.698887825012207, "length": 2.886237859725952, "mode": "train", "epoch": 17}
{"loss": 0.6002484560012817, "acc": 0.6876440048217773, "length": 3.1787710189819336, "mode": "test", "epoch": 17}
{"loss": 0.5614842772483826, "acc": 0.7246804237365723, "length": 2.9885668754577637, "mode": "train", "epoch": 18}
{"loss": 0.5633226633071899, "acc": 0.7245111465454102, "length": 3.1284916400909424, "mode": "test", "epoch": 18}
{"loss": 0.5171669721603394, "acc": 0.7473258972167969, "length": 3.296468496322632, "mode": "train", "epoch": 19}
{"loss": 0.5239192843437195, "acc": 0.7660722136497498, "length": 3.446927309036255, "mode": "test", "epoch": 19}
{"loss": 0.4717811048030853, "acc": 0.7844337821006775, "length": 3.4058375358581543, "mode": "train", "epoch": 20}
{"loss": 0.509403645992279, "acc": 0.7680951952934265, "length": 3.4804468154907227, "mode": "test", "epoch": 20}
{"loss": 0.4468384385108948, "acc": 0.8032733201980591, "length": 3.4410510063171387, "mode": "train", "epoch": 21}
{"loss": 0.5092998743057251, "acc": 0.7817599177360535, "length": 3.4804468154907227, "mode": "test", "epoch": 21}
{"loss": 0.42914634943008423, "acc": 0.8168021440505981, "length": 3.43255615234375, "mode": "train", "epoch": 22}
{"loss": 0.47895821928977966, "acc": 0.8146960735321045, "length": 3.5139665603637695, "mode": "test", "epoch": 22}
{"loss": 0.40169408917427063, "acc": 0.840345025062561, "length": 3.4646992683410645, "mode": "train", "epoch": 23}
{"loss": 0.4836784601211548, "acc": 0.8118277788162231, "length": 3.5642457008361816, "mode": "test", "epoch": 23}
{"loss": 0.40274345874786377, "acc": 0.8273693323135376, "length": 3.4657974243164062, "mode": "train", "epoch": 24}
{"loss": 0.4709230661392212, "acc": 0.8077403903007507, "length": 3.3966479301452637, "mode": "test", "epoch": 24}
{"loss": 0.378330796957016, "acc": 0.845207154750824, "length": 3.390153646469116, "mode": "train", "epoch": 25}
{"loss": 0.48103198409080505, "acc": 0.8081185817718506, "length": 3.581005573272705, "mode": "test", "epoch": 25}
{"loss": 0.3653280436992645, "acc": 0.8511200547218323, "length": 3.599835157394409, "mode": "train", "epoch": 26}
{"loss": 0.4449352025985718, "acc": 0.8191632032394409, "length": 3.5363128185272217, "mode": "test", "epoch": 26}
{"loss": 0.3557729721069336, "acc": 0.8532530069351196, "length": 3.5838825702667236, "mode": "train", "epoch": 27}
{"loss": 0.44957801699638367, "acc": 0.8010527491569519, "length": 3.7932960987091064, "mode": "test", "epoch": 27}
{"loss": 0.33967921137809753, "acc": 0.8577659130096436, "length": 3.6483285427093506, "mode": "train", "epoch": 28}
{"loss": 0.4415428638458252, "acc": 0.8265951871871948, "length": 3.614525079727173, "mode": "test", "epoch": 28}
{"loss": 0.32324323058128357, "acc": 0.8668872117996216, "length": 3.6923751831054688, "mode": "train", "epoch": 29}
{"loss": 0.45256420969963074, "acc": 0.8283256888389587, "length": 3.698323965072632, "mode": "test", "epoch": 29}
{"loss": 0.32642802596092224, "acc": 0.8708588480949402, "length": 3.8005013465881348, "mode": "train", "epoch": 30}
{"loss": 0.4842488765716553, "acc": 0.8177022933959961, "length": 3.698323965072632, "mode": "test", "epoch": 30}
{"loss": 0.3203960657119751, "acc": 0.8688614368438721, "length": 3.7098162174224854, "mode": "train", "epoch": 31}
{"loss": 0.43705877661705017, "acc": 0.832685649394989, "length": 3.6871509552001953, "mode": "test", "epoch": 31}
{"loss": 0.3023208975791931, "acc": 0.8766263127326965, "length": 3.754045009613037, "mode": "train", "epoch": 32}
{"loss": 0.47084176540374756, "acc": 0.8246318697929382, "length": 3.8994412422180176, "mode": "test", "epoch": 32}
{"loss": 0.2907432019710541, "acc": 0.8803273439407349, "length": 3.772193431854248, "mode": "train", "epoch": 33}
{"loss": 0.42692017555236816, "acc": 0.8504545092582703, "length": 3.614525079727173, "mode": "test", "epoch": 33}
{"loss": 0.2807225286960602, "acc": 0.8878984451293945, "length": 3.760056495666504, "mode": "train", "epoch": 34}
{"loss": 0.43162044882774353, "acc": 0.8341267108917236, "length": 3.7653632164001465, "mode": "test", "epoch": 34}
{"loss": 0.27810052037239075, "acc": 0.8872256875038147, "length": 3.782439708709717, "mode": "train", "epoch": 35}
{"loss": 0.41010090708732605, "acc": 0.8474223613739014, "length": 3.6480445861816406, "mode": "test", "epoch": 35}
{"loss": 0.2717435657978058, "acc": 0.887875497341156, "length": 3.764889717102051, "mode": "train", "epoch": 36}
{"loss": 0.41682732105255127, "acc": 0.8384084105491638, "length": 3.6592178344726562, "mode": "test", "epoch": 36}
{"loss": 0.26521167159080505, "acc": 0.8931068778038025, "length": 3.715860605239868, "mode": "train", "epoch": 37}
{"loss": 0.39618128538131714, "acc": 0.8483121395111084, "length": 3.664804458618164, "mode": "test", "epoch": 37}
{"loss": 0.2627328634262085, "acc": 0.8927431702613831, "length": 3.7691104412078857, "mode": "train", "epoch": 38}
{"loss": 0.37128257751464844, "acc": 0.8596016764640808, "length": 3.7430167198181152, "mode": "test", "epoch": 38}
{"loss": 0.2566094398498535, "acc": 0.8993856310844421, "length": 3.7862162590026855, "mode": "train", "epoch": 39}
{"loss": 0.40433168411254883, "acc": 0.8465861678123474, "length": 3.664804458618164, "mode": "test", "epoch": 39}
{"loss": 0.2488708198070526, "acc": 0.8964402675628662, "length": 3.743223190307617, "mode": "train", "epoch": 40}
{"loss": 0.389937162399292, "acc": 0.8441665172576904, "length": 3.720670461654663, "mode": "test", "epoch": 40}
{"loss": 0.2490711808204651, "acc": 0.9036564826965332, "length": 3.7773096561431885, "mode": "train", "epoch": 41}
{"loss": 0.42722779512405396, "acc": 0.8483259081840515, "length": 3.860335111618042, "mode": "test", "epoch": 41}
{"loss": 0.24043597280979156, "acc": 0.9007668495178223, "length": 3.792428970336914, "mode": "train", "epoch": 42}
{"loss": 0.437520831823349, "acc": 0.8461712002754211, "length": 3.6480445861816406, "mode": "test", "epoch": 42}
{"loss": 0.23787829279899597, "acc": 0.9075002670288086, "length": 3.788700819015503, "mode": "train", "epoch": 43}
{"loss": 0.4518052935600281, "acc": 0.8289781212806702, "length": 3.910614490509033, "mode": "test", "epoch": 43}
{"loss": 0.24623049795627594, "acc": 0.9053351283073425, "length": 3.8295249938964844, "mode": "train", "epoch": 44}
{"loss": 0.4172869622707367, "acc": 0.8484897613525391, "length": 3.782122850418091, "mode": "test", "epoch": 44}
{"loss": 0.22048333287239075, "acc": 0.9136995077133179, "length": 3.7678062915802, "mode": "train", "epoch": 45}
{"loss": 0.4217640161514282, "acc": 0.8446980714797974, "length": 3.692737340927124, "mode": "test", "epoch": 45}
{"loss": 0.2456488311290741, "acc": 0.9031122922897339, "length": 3.7504098415374756, "mode": "train", "epoch": 46}
{"loss": 0.38385626673698425, "acc": 0.8568620085716248, "length": 3.726257085800171, "mode": "test", "epoch": 46}
{"loss": 0.23100656270980835, "acc": 0.9084669947624207, "length": 3.7996020317077637, "mode": "train", "epoch": 47}
{"loss": 0.3582650423049927, "acc": 0.8592754602432251, "length": 3.754189968109131, "mode": "test", "epoch": 47}
{"loss": 0.21832303702831268, "acc": 0.9164742827415466, "length": 3.8107259273529053, "mode": "train", "epoch": 48}
{"loss": 0.36941203474998474, "acc": 0.8524667620658875, "length": 3.726257085800171, "mode": "test", "epoch": 48}
{"loss": 0.20154915750026703, "acc": 0.9226431250572205, "length": 3.7776200771331787, "mode": "train", "epoch": 49}
{"loss": 0.3691021800041199, "acc": 0.8600917458534241, "length": 3.720670461654663, "mode": "test", "epoch": 49}
{"loss": 0.200860396027565, "acc": 0.9211776256561279, "length": 3.803022861480713, "mode": "train", "epoch": 50}
{"loss": 0.38650697469711304, "acc": 0.8590625524520874, "length": 3.7653632164001465, "mode": "test", "epoch": 50}
{"loss": 0.20099785923957825, "acc": 0.9226324558258057, "length": 3.8228676319122314, "mode": "train", "epoch": 51}
{"loss": 0.41106289625167847, "acc": 0.859203577041626, "length": 3.726257085800171, "mode": "test", "epoch": 51}
{"loss": 0.19908659160137177, "acc": 0.9221848249435425, "length": 3.805346727371216, "mode": "train", "epoch": 52}
{"loss": 0.3822111487388611, "acc": 0.8616108894348145, "length": 3.748603343963623, "mode": "test", "epoch": 52}
{"loss": 0.18894517421722412, "acc": 0.9266951680183411, "length": 3.8071515560150146, "mode": "train", "epoch": 53}
{"loss": 0.3752039074897766, "acc": 0.8655864596366882, "length": 3.770949602127075, "mode": "test", "epoch": 53}
{"loss": 0.18615861237049103, "acc": 0.9294473528862, "length": 3.824586868286133, "mode": "train", "epoch": 54}
{"loss": 0.3712945580482483, "acc": 0.8670658469200134, "length": 3.7877094745635986, "mode": "test", "epoch": 54}
{"loss": 0.19002841413021088, "acc": 0.9273965954780579, "length": 3.8338260650634766, "mode": "train", "epoch": 55}
{"loss": 0.3811734914779663, "acc": 0.8688682913780212, "length": 3.770949602127075, "mode": "test", "epoch": 55}
{"loss": 0.2044130563735962, "acc": 0.9208876490592957, "length": 3.818565607070923, "mode": "train", "epoch": 56}
{"loss": 0.3721843659877777, "acc": 0.862943172454834, "length": 3.7094972133636475, "mode": "test", "epoch": 56}
{"loss": 0.18242740631103516, "acc": 0.9293145537376404, "length": 3.79622483253479, "mode": "train", "epoch": 57}
{"loss": 0.3748353123664856, "acc": 0.8619140982627869, "length": 3.7653632164001465, "mode": "test", "epoch": 57}
{"loss": 0.1669079214334488, "acc": 0.9359815120697021, "length": 3.832352876663208, "mode": "train", "epoch": 58}
{"loss": 0.3515324294567108, "acc": 0.8713706135749817, "length": 3.81005597114563, "mode": "test", "epoch": 58}
{"loss": 0.17215169966220856, "acc": 0.928538978099823, "length": 3.8523190021514893, "mode": "train", "epoch": 59}
{"loss": 0.4202200770378113, "acc": 0.8548022508621216, "length": 3.804469347000122, "mode": "test", "epoch": 59}
{"loss": 0.17242319881916046, "acc": 0.9302762746810913, "length": 3.8521716594696045, "mode": "train", "epoch": 60}
{"loss": 0.38314613699913025, "acc": 0.8656615614891052, "length": 3.782122850418091, "mode": "test", "epoch": 60}
{"loss": 0.18829026818275452, "acc": 0.9311228394508362, "length": 3.828423500061035, "mode": "train", "epoch": 61}
{"loss": 0.4095315933227539, "acc": 0.8651822209358215, "length": 3.748603343963623, "mode": "test", "epoch": 61}
{"loss": 0.18662823736667633, "acc": 0.9293471574783325, "length": 3.793816566467285, "mode": "train", "epoch": 62}
{"loss": 0.404281347990036, "acc": 0.8619325160980225, "length": 3.7597765922546387, "mode": "test", "epoch": 62}
{"loss": 0.17886710166931152, "acc": 0.9311308264732361, "length": 3.832310914993286, "mode": "train", "epoch": 63}
{"loss": 0.40662312507629395, "acc": 0.8491880893707275, "length": 3.83798885345459, "mode": "test", "epoch": 63}
{"loss": 0.18373577296733856, "acc": 0.9308903217315674, "length": 3.8609461784362793, "mode": "train", "epoch": 64}
{"loss": 0.38074108958244324, "acc": 0.8693981766700745, "length": 3.7932960987091064, "mode": "test", "epoch": 64}
{"loss": 0.1658392697572708, "acc": 0.9400397539138794, "length": 3.8363888263702393, "mode": "train", "epoch": 65}
{"loss": 0.35705146193504333, "acc": 0.8672833442687988, "length": 3.8156423568725586, "mode": "test", "epoch": 65}
{"loss": 0.1675669103860855, "acc": 0.9346454739570618, "length": 3.8255598545074463, "mode": "train", "epoch": 66}
{"loss": 0.3745618760585785, "acc": 0.8749127984046936, "length": 3.770949602127075, "mode": "test", "epoch": 66}
{"loss": 0.15297484397888184, "acc": 0.9437191486358643, "length": 3.8331756591796875, "mode": "train", "epoch": 67}
{"loss": 0.3573765456676483, "acc": 0.878716766834259, "length": 3.7318434715270996, "mode": "test", "epoch": 67}
{"loss": 0.15517066419124603, "acc": 0.9419080018997192, "length": 3.815203905105591, "mode": "train", "epoch": 68}
{"loss": 0.35925862193107605, "acc": 0.8739677667617798, "length": 3.7932960987091064, "mode": "test", "epoch": 68}
{"loss": 0.14827829599380493, "acc": 0.9423707723617554, "length": 3.8443450927734375, "mode": "train", "epoch": 69}
{"loss": 0.38710182905197144, "acc": 0.8679417371749878, "length": 3.770949602127075, "mode": "test", "epoch": 69}
{"loss": 0.1594032496213913, "acc": 0.9403549432754517, "length": 3.812333345413208, "mode": "train", "epoch": 70}
{"loss": 0.38976365327835083, "acc": 0.8723782896995544, "length": 3.7877094745635986, "mode": "test", "epoch": 70}
{"loss": 0.1538972705602646, "acc": 0.9396438002586365, "length": 3.8352620601654053, "mode": "train", "epoch": 71}
{"loss": 0.35601264238357544, "acc": 0.8825376033782959, "length": 3.754189968109131, "mode": "test", "epoch": 71}
{"loss": 0.15286995470523834, "acc": 0.9414865374565125, "length": 3.833463668823242, "mode": "train", "epoch": 72}
{"loss": 0.36434900760650635, "acc": 0.8743368983268738, "length": 3.8715083599090576, "mode": "test", "epoch": 72}
{"loss": 0.1804766058921814, "acc": 0.9325515627861023, "length": 3.8922195434570312, "mode": "train", "epoch": 73}
{"loss": 0.39327067136764526, "acc": 0.8741883635520935, "length": 3.83798885345459, "mode": "test", "epoch": 73}
{"loss": 0.17670312523841858, "acc": 0.9369663000106812, "length": 3.8564963340759277, "mode": "train", "epoch": 74}
{"loss": 0.33888572454452515, "acc": 0.8857245445251465, "length": 3.7877094745635986, "mode": "test", "epoch": 74}
{"loss": 0.16170133650302887, "acc": 0.9414282441139221, "length": 3.8493258953094482, "mode": "train", "epoch": 75}
{"loss": 0.3709535598754883, "acc": 0.8825820088386536, "length": 3.83798885345459, "mode": "test", "epoch": 75}
{"loss": 0.14686520397663116, "acc": 0.9427580833435059, "length": 3.8644535541534424, "mode": "train", "epoch": 76}
{"loss": 0.4054102897644043, "acc": 0.8638559579849243, "length": 3.804469347000122, "mode": "test", "epoch": 76}
{"loss": 0.20292861759662628, "acc": 0.9251693487167358, "length": 3.8377528190612793, "mode": "train", "epoch": 77}
{"loss": 0.36532410979270935, "acc": 0.8771746754646301, "length": 3.748603343963623, "mode": "test", "epoch": 77}
{"loss": 0.17645426094532013, "acc": 0.9310165643692017, "length": 3.80183744430542, "mode": "train", "epoch": 78}
{"loss": 0.3726121187210083, "acc": 0.8720659017562866, "length": 3.7932960987091064, "mode": "test", "epoch": 78}
{"loss": 0.21434833109378815, "acc": 0.9232557415962219, "length": 3.8287606239318848, "mode": "train", "epoch": 79}
{"loss": 0.40846309065818787, "acc": 0.8575389981269836, "length": 3.776536226272583, "mode": "test", "epoch": 79}
{"loss": 0.18984371423721313, "acc": 0.9287954568862915, "length": 3.8162357807159424, "mode": "train", "epoch": 80}
{"loss": 0.3754502534866333, "acc": 0.8809435367584229, "length": 3.776536226272583, "mode": "test", "epoch": 80}
{"loss": 0.16714078187942505, "acc": 0.9346351027488708, "length": 3.836393356323242, "mode": "train", "epoch": 81}
{"loss": 0.3812856078147888, "acc": 0.8740965127944946, "length": 3.7932960987091064, "mode": "test", "epoch": 81}
{"loss": 0.15153229236602783, "acc": 0.9417253136634827, "length": 3.847017765045166, "mode": "train", "epoch": 82}
{"loss": 0.4029158055782318, "acc": 0.8692466020584106, "length": 3.7932960987091064, "mode": "test", "epoch": 82}
{"loss": 0.16481655836105347, "acc": 0.9364292621612549, "length": 3.845022678375244, "mode": "train", "epoch": 83}
{"loss": 0.412350058555603, "acc": 0.870643138885498, "length": 3.804469347000122, "mode": "test", "epoch": 83}
{"loss": 0.14349475502967834, "acc": 0.9421421885490417, "length": 3.8572804927825928, "mode": "train", "epoch": 84}
{"loss": 0.37263476848602295, "acc": 0.8788515329360962, "length": 3.81005597114563, "mode": "test", "epoch": 84}
{"loss": 0.14289996027946472, "acc": 0.9473408460617065, "length": 3.8608458042144775, "mode": "train", "epoch": 85}
{"loss": 0.39931386709213257, "acc": 0.8681010603904724, "length": 3.81005597114563, "mode": "test", "epoch": 85}
{"loss": 0.14384515583515167, "acc": 0.9446559548377991, "length": 3.8590171337127686, "mode": "train", "epoch": 86}
{"loss": 0.3741781711578369, "acc": 0.8776218891143799, "length": 3.8212289810180664, "mode": "test", "epoch": 86}
{"loss": 0.13525359332561493, "acc": 0.9486136436462402, "length": 3.858860731124878, "mode": "train", "epoch": 87}
{"loss": 0.34323346614837646, "acc": 0.8855959177017212, "length": 3.8156423568725586, "mode": "test", "epoch": 87}
{"loss": 0.13660763204097748, "acc": 0.9492313265800476, "length": 3.860596179962158, "mode": "train", "epoch": 88}
{"loss": 0.3452061116695404, "acc": 0.8893692493438721, "length": 3.81005597114563, "mode": "test", "epoch": 88}
{"loss": 0.13399192690849304, "acc": 0.9422111511230469, "length": 3.8548452854156494, "mode": "train", "epoch": 89}
{"loss": 0.3562142848968506, "acc": 0.881591260433197, "length": 3.804469347000122, "mode": "test", "epoch": 89}
{"loss": 0.12066702544689178, "acc": 0.953925371170044, "length": 3.840869188308716, "mode": "train", "epoch": 90}
{"loss": 0.3516826033592224, "acc": 0.8901563286781311, "length": 3.7597765922546387, "mode": "test", "epoch": 90}
{"loss": 0.11923009902238846, "acc": 0.9536886811256409, "length": 3.841585159301758, "mode": "train", "epoch": 91}
{"loss": 0.3761940896511078, "acc": 0.8757534623146057, "length": 3.81005597114563, "mode": "test", "epoch": 91}
{"loss": 0.11438282579183578, "acc": 0.9533993005752563, "length": 3.8574509620666504, "mode": "train", "epoch": 92}
{"loss": 0.3442099094390869, "acc": 0.8878960013389587, "length": 3.8156423568725586, "mode": "test", "epoch": 92}
{"loss": 0.11584344506263733, "acc": 0.9565016031265259, "length": 3.853813648223877, "mode": "train", "epoch": 93}
{"loss": 0.37130090594291687, "acc": 0.8796800374984741, "length": 3.804469347000122, "mode": "test", "epoch": 93}
{"loss": 0.1165430098772049, "acc": 0.9523245692253113, "length": 3.8497495651245117, "mode": "train", "epoch": 94}
{"loss": 0.35370904207229614, "acc": 0.8836755156517029, "length": 3.804469347000122, "mode": "test", "epoch": 94}
{"loss": 0.11980396509170532, "acc": 0.9543331861495972, "length": 3.8532280921936035, "mode": "train", "epoch": 95}
{"loss": 0.35860511660575867, "acc": 0.8871182203292847, "length": 3.782122850418091, "mode": "test", "epoch": 95}
{"loss": 0.14332054555416107, "acc": 0.9464390873908997, "length": 3.846158027648926, "mode": "train", "epoch": 96}
{"loss": 0.4089173674583435, "acc": 0.8835146427154541, "length": 3.782122850418091, "mode": "test", "epoch": 96}
{"loss": 0.1453026682138443, "acc": 0.9449120163917542, "length": 3.849931478500366, "mode": "train", "epoch": 97}
{"loss": 0.3781435489654541, "acc": 0.880713701248169, "length": 3.770949602127075, "mode": "test", "epoch": 97}
{"loss": 0.14258666336536407, "acc": 0.9434707164764404, "length": 3.834881544113159, "mode": "train", "epoch": 98}
{"loss": 0.40894609689712524, "acc": 0.8853157758712769, "length": 3.7988827228546143, "mode": "test", "epoch": 98}
{"loss": 0.13860948383808136, "acc": 0.9462430477142334, "length": 3.8659329414367676, "mode": "train", "epoch": 99}
{"loss": 0.3270286023616791, "acc": 0.8960232138633728, "length": 3.804469347000122, "mode": "test", "epoch": 99}
{"loss": 0.1244024783372879, "acc": 0.9540557861328125, "length": 3.84564471244812, "mode": "train", "epoch": 100}
{"loss": 0.3245006203651428, "acc": 0.8988456726074219, "length": 3.776536226272583, "mode": "test", "epoch": 100}
test accuracy: 0.8381496667861938
